patches/api/0172-Fix-Spigot-annotation-mistakes.patch

```diff
@@ -510,6 +510,19 @@ index e1123295b9511a2c610a1baf7195638f7f3e64c4..273ae8e5da0a858d3b82d1b0f5992318
      @Override
      public void setCancelled(boolean cancel) {
          this.cancel = cancel;
+diff --git a/src/main/java/org/bukkit/event/inventory/InventoryClickEvent.java b/src/main/java/org/bukkit/event/inventory/InventoryClickEvent.java
+index fe58058f9b5d29388d48115cc81dc48ab08c58c1..ac67a4e321f43d1ede09dafe2daa1f07de8a923f 100644
+--- a/src/main/java/org/bukkit/event/inventory/InventoryClickEvent.java
++++ b/src/main/java/org/bukkit/event/inventory/InventoryClickEvent.java
+@@ -85,7 +85,7 @@ public class InventoryClickEvent extends InventoryInteractEvent {
+      *
+      * @return the cursor ItemStack
+      */
+-    @Nullable
++    @NotNull // Paper - fix nullability
```

Comment:
These changes should already be in that patch?

Reply:
What I'm implying is that you may consider [modifying](https://github.com/PaperMC/Paper/blob/master/CONTRIBUTING.md#modifying-patches) said patch with those annotation changes instead of having them in this one.
Not sure if this is required since the team already approved the changes tho.

Reply:
tell me the name of the patch the changes are right now.
and tell me the name of the patch you want them to be.